src/components/ProgressBar/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC } from 'react'
+
+import { Typography } from '../Typography'
+import s from './core/progressbar.module.scss'
+import { IProps } from './core/progressbar.typings'
+
+export const ProgressBar: FC<IProps> = ({ percentage }) => {
+  return (
+    <div className={s['container']}>
+      <Typography>{percentage}%</Typography>
```

Comment:
If we use the `<progress>` element, the variables in scope would be `max` and `value`. 

Passing the percentage would be overkill as all the info needed to compute it here is available.

```suggestion
      <Typography>{(100 * props.value) / props.max }%</Typography>
```

Or extracted as a function

```ts
const percentage = (value: number, max: number) => (100 * value) / max
```